### DISCUSSION TITLE
Where can I download the latest Primeblocks Angular project?

### DISCUSSION THREAD

**Discussion by morsagmon** [primefaces/community#1560] [PrimeNG Blocks] [answered]
I purchased the Primeblocks life license and installed version 3.2.1 at the time.
Where can I download the latest Angular project for Primeblocks?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

You can download them from https://www.primefaces.org/store 
Please e-mail contact@primetek.com.tr if you don't have a Prime Store account yet.

**Reply by morsagmon** [reply]
Hi,
Thank you for the link!
I noticed there are new blocks in the demo website available since I purchased back in August 2023.
However, in the store, the latest version I can see for download is 3.2.1 - the same version I already installed back in August.
Is this still the most up-to-date version available? how can I enjoy the new blocks that were added since then?

Thanks!
